### ISSUE TITLE
add sample pdfs

### ISSUE THREAD

**Issue by clarkewd** [plessl/wkpdf#30]
Please add sample pdfs to either the homepage or the repo. This looks like a nice utility but there are no screenshots or example files (that i could find) in this repo or on the info page [http://plessl.github.com/wkpdf/](http://plessl.github.com/wkpdf/)

**Comment by plessl** [comment]
Thanks for the suggestion, but I don't understand what you have in mind. What are the screenshots supposed to show? wkpdf is a command line utility hence the screenshot will be boring. An the generated PDFs will look exactly like what is generated when printing the webpage to PDF in Safari.

Do you have a suggestion how to visualize what wkpdf is doing in a nice graphical way?

**Comment by clarkewd** [comment]
This was _not_ meant as a slight. I believe people who are considering using software like to immediately "see" what it looks like. I usually look for screenshots when I am visiting an webpage for something. I think even if you added some screenshots of typing the command in the terminal window, or perhaps a desktop with a terminal window, a browser, and then the rendered image in the `preview.app` - it would just get the message across in a very immediate fashion.

Regarding the sample PDF files - I still think it would be nice if you could render a few webpages and make the PDF files available for download. Do the GitHub / Apple / CNN homepages and post those. Yes it should look "exactly" like the webpage in safari but anyone who has been exploring PDF rendering options knows about all the challenges - fonts, spacing, paper size, etc. I don't want to have to download / install a program just to see if I like the way it renders PDF files, so how about adding some generated sample files?

Basically it looks like you have a good thing going here, and I think for it to really "thrive" it could use a larger following (people reporting bugs, people helping with development, etc) - so these are my suggestions on getting more people to use it.

**Comment by plessl** [comment]
Thanks for elaborating on your idea. I agree that presenting potential users with a nicer illustration of what wkpdf is capable of would be good thing. I will put it on my TODO list.
